preview/Dockerfile

```diff
@@ -1,5 +1,11 @@
 FROM buildpack-deps:trusty-scm
 
+# Work around dotnet/cli#1582 until Docker releases a fix (docker/docker#20818).
```

Comment:
I didn't intend to use the abbreviated github link syntax.  I think github shortened what I had typed into my comment.  I think the Dockerfile should use the real link.